src/main/java/org/spongepowered/common/bridge/world/inventory/container/TrackedContainerBridge.java

```diff
@@ -28,5 +28,5 @@ public interface TrackedContainerBridge {
 
     boolean bridge$capturePossible();
 
-    void bridge$detectAndSendChanges(boolean captureOnly);
+    void bridge$detectAndSendChanges(boolean capture, boolean synchronize);
```

Comment:
Just to be clear, the synchronize argument is to synchronize any changes back to the client, yes?

Reply:
Yes. If true, it will send the carried item and data slots to the client, but `broadcastFullState` already do that so it doesn't need to do it again.